packages/dashboard/src/components/Card/Layout/Layout.tsx

```diff
@@ -174,32 +178,29 @@ export class Layout extends React.Component<ILayoutProps> {
     );
   }
 
-  private _generateContentArea(
-    cardContentList: ICardContentDetails[],
-    contentLayoutClassName: string,
-    contentArea1ClassName: string,
-    dataVizLastUpdateClassName: string,
-    contentArea2ClassName: string,
-    cardSize: CardSize
-  ): JSX.Element | null {
+  private _generateContentArea(cardContentList: ICardContentDetails[], cardSize: CardSize, header?: ICardHeaderProps): JSX.Element | null {
     if (cardContentList === null || cardContentList === undefined) {
       return null;
     }
-
-    const contentAreaContents = this._generateContentElement(cardContentList, dataVizLastUpdateClassName);
+    const getClassNames = classNamesFunction<ILayoutStyleProps, ILayoutStyles>();
```

Comment:
See above - should be at the top of the file.

Reply:
updated